packages/tools/src/tools/annotation/PlanarFreehandContourSegmentationTool.ts

```diff
@@ -10,6 +10,7 @@ class PlanarFreehandContourSegmentationTool extends PlanarFreehandROITool {
       {
         configuration: {
           calculateStats: false,
+          allowOpenContours: false,
```

Comment:
This causes the contours to be drawn closed even during initial creation.

Reply:
can you handle others as well. Also I'm confused since there is another configuration option called `closed`, what is that?